app/src/main/java/org/simple/clinic/patient/businessid/BusinessId.kt

```diff
@@ -84,17 +85,20 @@ data class BusinessId(
   sealed class MetaVersion {
 
     companion object {
-      @VisibleForTesting
+      @VisibleForTesting(otherwise = VisibleForTesting.NONE)
       fun random() = TypeAdapter.knownMappings.keys.shuffled().first()
+
+      @VisibleForTesting(otherwise = VisibleForTesting.NONE)
+      fun values() = TypeAdapter.knownMappings.keys
     }
 
-    object BpPassportMetaV1 : MetaVersion()
+    object BpPassportMetaVersionV1 : MetaVersion()
```

Comment:
`VersionV1`? V=Version, so can we rename this to something better? Maybe leave it as V1 or Version1?

Reply:
Might also require commit reword

Reply:
`VersionV1` 😂

Reply:
> `VersionV1` 😂

This is intentional, since we have an enum `meta_version` which represents the type of metadata that is serialised in the `BusinessId` model.

So a model of type `BpPassport` might have three different types of encoded metadata, `v1`, `v2`, `v3`, which will be represent as `BpPassportMetaVersionV1`, `BpPassportMetaVersionV2`, `BpPassportMetaVersionV3` and so on.

Reply:
I have renamed the enum values to remove the `Version` string. This makes the enum values a little easier to read, but it also means that statically importing both enum types of `BusinessIdMeta` and `BusinessIdMetaVersion` in a single file is not possible since the values in both enums are not `BpPassportV1`.